osquery/tables/system/posix/augeas.cpp

```diff
@@ -121,7 +121,10 @@ class AugeasHandle {
   void initialize() {
     std::call_once(initialized, [this]() {
       this->aug = aug_init(
-          nullptr, FLAGS_augeas_lenses.c_str(), AUG_NO_ERR_CLOSE | AUG_NO_LOAD);
+          nullptr,
+          FLAGS_augeas_lenses.c_str(),
+          AUG_NO_ERR_CLOSE | AUG_NO_LOAD | AUG_NO_STDINC | AUG_SAVE_NOOP);
```

Comment:
Adding `AUG_NO_STDINC` is a somewhat breaking change. It will cause augeas to stop loading the system provided lenses. I think this is correct.